openbas-api/src/main/java/io/openbas/rest/payload/form/PayloadCreateInput.java

```diff
@@ -28,11 +30,11 @@ public class PayloadCreateInput {
 
   @NotBlank(message = MANDATORY_MESSAGE)
   @JsonProperty("payload_source")
-  private String source;
+  private PAYLOAD_SOURCE source;
```

Comment:
I have an internal error when I try to create a payload
![Uploading image.png…]()